release-notes/ugandaemr-337.md

```diff
@@ -0,0 +1,66 @@
+# 3.3.7 - March 2022 
+
+## New Features
+
+1. Martnernal and Child Health
+  
+   * Antenatal  Services for both Point of Care and Restrospective approach. 
+   * Maternity Services
+   * Postnatal Services
+   * Reports - (105 MCH Section, ANC 105, Marternity 105, 105 PNC )
+   
+2. Safe Male Circumcision
+
+    *Retrospective Data Entry for SMC Services
+    
+3. Create a generic approach to generating  and Sync FHIR Encounters for data exchange
+4. Added Missing Districts
+    
+## Improvements
+ 
+ 1. Add Display Board for Logs
```

Comment:
Which results, Viral Load?

Reply:
VL, ART Access can be monitored, if possible, add recency. I think all these have timestamps of successful data sync in the maintenance section, but as we have seen over time while troubleshooting recency, the timestamp will report the successful sync by the system but will not inform the users that the payload was blank or got an error. I think this board should be used to address that.